### ISSUE TITLE
Composer install fails

### ISSUE THREAD

**Issue by gavinhewitt** [LeanAdmin/ecom-example#4]
Getting this error:

```
[RuntimeException]                                                                      
Source path "/Users/samuel/Projects/lean/lean" is not found for package leanadmin/lean
```

**Comment by stancl** [comment]
https://github.com/LeanAdmin/ecom-example#installation

Try step 3 in here.

Also, the demo is not updated for the latest version I think. I'll be pushing a new version soon — I've been using it in dev for new features like darkmode (just added), data tables (coming in the next few days), CSS customization, etc.

**Comment by gavinhewitt** [comment]
Yes, that's what I did, and then `composer install` (step 4) which breaks.

**Comment by stancl** [comment]
Ah. Try deleting `composer.lock` and `vendor/` and running `composer install` again.

**Comment by gavinhewitt** [comment]
That worked, but now getting this:

```
PHP Fatal error:  Type of App\Lean\Resources\CustomerResource::$resultsPerPage must be int (as in class Lean\LeanResource) in /Users/gavin/Projects/ecom-example/app/Lean/Resources/CustomerResource.php on line 0
```

Gonna check if there's an easy fix

**Comment by stancl** [comment]
Ah. Must be the new versions.

I could push my local version but it has some debugging calls in it. I'll try to update the demo tomorrow morning.

**Comment by gavinhewitt** [comment]
Fixed them by removing from resources. But now step 4+5 give me this:

```
  Database (ecom_lean) does not exist. (SQL: PRAGMA foreign_keys = ON;)
```

I give up. The online demo is currently not working. Is that the demo you are referring to? Would be good to see what CreateOrderAction does/looks like.

**Comment by stancl** [comment]
Going to bed right now but if you want to *see* it, it's on the project's landing page.

Basically when you click Create Order you get taken to that custom page instead of the default create screen.

I'll be setting up a new demo soon as well.

**Comment by gavinhewitt** [comment]
Ah yes, I see it. Thanks.

But for my understanding, there's now no way to add tags based on BelongsToMany to a resource. For that you'd need to make something custom, most likely an action. But actions only cover crud operations, not custom operations. So could you give some insight on how that might work?

I'm also thinking something like a page where you can reorder resources (drag and drop). 

And will you be implementing something like bulk actions, similar to Nova? I have the use case where I need to tag multiple resources, with multiple tags. Not sure how that would work in Lean.

**Comment by gavinhewitt** [comment]
>And will you be implementing something like bulk actions, similar to Nova? I have the use case where I need to tag multiple resources, with multiple tags. Not sure how that would work in Lean.

Regarding this, never mind. Joined Discord and saw it's a todo

**Comment by stancl** [comment]
Bulk actions will be both Nova-style and bulk edit ideally

**Comment by rslucena** [comment]
Reopening the topic.

[Composer\Downloader\TransportException]
The "https://api.github.com/repos/LeanAdmin/lean" file could not be downloaded (HTTP/2 404 ) 

I have already done all the steps for the installation, but it is not possible to pass this step. 

*It asked me for an access token too, if I don't add any it interrupts, creating a token in oauth, it still kept giving the problem

**Comment by stancl** [comment]
Do you have access to the Lean repo?

**Comment by rslucena** [comment]
How can I make this request? I would like to use the demo to test the package

**Comment by stancl** [comment]
The demo requires access to the package, which requires a license.

It's not a demo to try the package before buying, it's for seeing how it can be used after you already have a license.

Either way, it's way outdated right now. I'll be making a new demo at v1 launch.
